WeatherForecast/WeatherForecast/Resources/Extensions.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Extensions.swift
+//  WeatherForecast
+//
+//  Created by 윤형석 on 1/30/24.
+//
+
+import Foundation
+import UIKit
+
+extension Date {
+    func toString(format: String) -> String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.locale = .init(identifier: "ko_KR")
+        dateFormatter.dateFormat = format
+        return dateFormatter.string(from: self)
+    }
+    
+    func toString(timeStyle: DateFormatter.Style) -> String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.locale = .init(identifier: "ko_KR")
+        dateFormatter.timeStyle = .short
+        return dateFormatter.string(from: self)
+    }
+}
+
+extension Data {
+    func toUIImage() -> UIImage? {
```

Comment:
```suggestion
    var uiImage: UIImage? {
```

 p4;

매개변수를 받아올 필요가 없다면 연산 프로퍼티로 작성해보면 어떨까요?